src/components/emotes.ts

```diff
@@ -0,0 +1,8 @@
+import { Emoji } from 'discord.js';
+import { CommandoClient } from 'discord.js-commando';
+
+export const emoteList: { [key: string]: Emoji } = {};
+
+export const initEmotes = (client: CommandoClient): void => {
+  client.emojis.cache.forEach((emoji) => (emoteList[emoji.name] = emoji));
```

Comment:
Can we also stick with one naming also? `Emoji` vs `Emote`. I think Discord uses emoji, so maybe we'd want to as well?